src/content_sao.cpp

```diff
@@ -1,6 +1,7 @@
 /*
 Minetest
 Copyright (C) 2010-2013 celeron55, Perttu Ahola <celeron55@gmail.com>
+(C) 2018 crazyBaboon, nuno360@gmail.com
```

Comment:
You literally changed like 3 lines in this file (static cache, if start, if end). There are 38 other contributors to this file who have done more

Reply:
I believe it is better to add copyright notice to the files or else it might get lost. If there are one million authors, move the copyright notice (or part of it) to the bottom of the file.

Reply:
Git blame is very reliable to find the author of a changed line.

Reply:
The other authors mostly didn't add their names because they didn't do anything significant enough to justify it. Those who do change a lot often don't bother to add their names either.
We only add authors if they do something very significant to a file, so we can't add your name as it then gives an unbalanced impression.